Sources/Danger/GitHubDSL.swift

```diff
@@ -201,7 +201,7 @@ extension GitHub {
         }
 
         /// The UUID for the user organization.
-        public let id: Int // swiftlint:disable:this identifier_name
+        public let id: Int 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 204;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)